spark/lib/oxygen/physicsserver/dragcontroller_c.cpp

```diff
@@ -65,11 +65,57 @@ FUNCTION(DragController,setAngularDrag)
         return true;
 }
 
+FUNCTION(DragController,getStillness)
+{
+    return obj->GetStillness();
+}
+
+FUNCTION(DragController,setStillness)
+{
+    float inDrag;
```

Comment:
i feel like this should be `inStillness` to decrease confusion

Reply:
Ah. The classic copy pasta XD

Fixed